### PR TITLE
Upgrade ONNX to 1.13.0

### PR DESCRIPTION
**Pull request by hamptonm1**
Upgraded onnx to 1.13.0 which corresponds to commit number `1ba78561`
I updated the documentation by running a `make onnx-mlir-docs`
I also updated `src/Dialect/ONNX/ONNXOps.td.inc` by issuing a `make OMONNXOpsIncTranslation`
The version number in `utils/gen_onnx_mlir.py` has also been changed to 1.13.0.
Please let me know if this looks good to you all. Thanks!!

### CODE REVIEW

**Comment by hamptonm1** [comment]
@philass and myself somewhat divided the work for the  onnx upgrade. He will do the remaining pieces and I will shadow his efforts so I know what needs to be done in the near future. 

FYI- @cjvolzka

**Comment by chentong319** [comment]
Could you  open an issue to list out the work items for Ops of  ONNX 1.13.0 so that the onnx-mlir community are aware of them?

**Review by chentong319 — APPROVED** [review]
LGTM!

**Review by philass — APPROVED** [review]
LGTM! Thanks for doing this @hamptonm1

I'll make sure to comment back in the tests that need to be tweaked for opset 18 ops in my following PR

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Detailed instructions are provided below.
 ```
 python >= 3.8
 gcc >= 6.4
-protobuf >= 3.18.3
+protobuf >= 3.20.2
```
Comment (cjvolzka) [inline README.md:53]: There was a CVE for 3.20.2 so we should move all the way to 3.20.3 instead.

Comment (hamptonm1) [reply README.md:53]: Thanks... I can do that!

**Review by AlexandreEichenberger — COMMENTED** [review]
Can you attach a list of which tasks this PR did in order to update to the latest? Esp with respect to existing ops that changed? Does it cover all/none/some of them? Will be useful to track the remaining tasks. Tx for this important work

**Comment by hamptonm1** [comment]
> Can you attach a list of which tasks this PR did in order to update to the latest? Esp with respect to existing ops that changed? Does it cover all/none/some of them? Will be useful to track the remaining tasks. Tx for this important work

Yes I can do that!

**Comment by hamptonm1** [comment]
Thank you so much @gongsu832 for finding the culprit to why protobuf was not being found during the docker build of `onnx-mlir`. Seems like protobuf was being installed in `/usr/local/lib` but the code was looking for `/usr/local/lib64`. So the docker file had the wrong path! Thank you again for all your help!!! I updated the docker files with the code snippet we discussed :)

**Comment by hamptonm1** [comment]
> Can you attach a list of which tasks this PR did in order to update to the latest? Esp with respect to existing ops that changed? Does it cover all/none/some of them? Will be useful to track the remaining tasks. Tx for this important work

This PR updated the following (please also look at the description for more details @AlexandreEichenberger :

1. Upgraded onnx to 1.13.0
2. Commented out the failing `backend test` for now (@philass will contribute to the second half of this upgrade and uncomment out the test)
3. Updated the docker files so that protobuf is found in the correct path.
4. Upgraded `protobuf` from `3.18.3` to `3.20.3`

Let me know if this list looks good to you?

**Pull request merged**